src/tests/mapUtils.test.ts

```
@@ -1,7 +1,13 @@
 import { LayerInfo } from '../data/mapData'
-import { mapRegionSelected, getActiveLayers } from '../utils/mapUtils'
+import {
+  mapRegionSelected,
+  getActiveLayers,
+  createPolygonHoverHandler,
+  createPolygonClickHandler,
+} from '../utils/mapUtils'
 import { RegionOption } from '../types/RegionDataTypes'
-import { MapGeoJSONFeature, LngLat } from 'maplibre-gl'
+import { MapLayerMouseEvent, MapGeoJSONFeature, LngLat, GeoJSONFeature } from 'maplibre-gl'
```

ESLint

[error] 'MapLayerMouseEvent' is defined but never used.

(@typescript-eslint/no-unused-vars)